taxi/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver
+
+
+class DriverLicenceUpdateForm(forms.ModelForm):
```

Comment:
add DriverCreationForm with additional fields "first_name", "last_name" and "license_number". When you create a user `license_number` must be validated as well